src/__tests__/app.integration.test.js

```diff
@@ -146,9 +173,9 @@ describe('POST /registration-requests/', () => {
 
     process.env.SERVICE_URL = serviceUrl;
     process.env.API_KEY_FOR_TEST = fakeAuth;
-    process.env.GP2GP_MESSENGER_SERVICE_URL = localhostUrl;
+    process.env.GP2GP_MESSENGER_SERVICE_URL = EHR_OUT;
     process.env.GP2GP_MESSENGER_AUTHORIZATION_KEYS = fakeAuth;
-    process.env.EHR_REPO_SERVICE_URL = localhostUrl;
+    process.env.EHR_REPO_SERVICE_URL = EHR_OUT;
```

Comment:
is it good to name the localhost as EHR_OUT ?  Because this string would be used to mock both GP2GP messenger and EHR-REPO. 
New comer reading this may found it confusing that we mock multiple endpoints named "EHR_OUT", but those endpoints are actually meant to be GP2GP messenger and EHR repo respectively.

Reply:
I agree! Though this is outside the scope of 3162 I would argue, this was implemented in a previous ticket.